services/timeseries.py

```diff
@@ -227,17 +228,23 @@ def repository_flag_ids(repository: Repository) -> Mapping[str, int]:
     return {repo_flag.flag_name: repo_flag.id for repo_flag in repo_flags}
 
 
```

Comment:
Before this change the batch size is always determined by number of flags even if the repo has no flags. Now it will batch by number of flags if trying to backfill flag coverage, batch by number of components if trying to backfill component coverage, and don't further batch (ie default to `backfill_max_batch_size`) if trying to backfill by generic coverage.